### ISSUE TITLE
check_modbus

### ISSUE THREAD

**Issue by AndreySV** [leshcat/leshcatlabs#1]
Hi, 

I've found your blogpost on the Internet.
http://leshcatlabs.net/2014/04/15/powerlogic-egx300-monitoring-with-nagios/
I want to ask what kind of issiues did you have with check_modbus and nagios?  If the check_modbus's output is wrong according to Nagios standards, that it's a bug and I'd like to fix that.

**Comment by leshcat** [comment]
Hi there,

I've actually didn't have any problems with output. 

Everything worked as expected. I've just made my own bash wrapper to suit my needs.

That's pretty much it :)

Thank you for making this great application, w/o it i would not be able to resolve the task i was given.

Regards,
Dzenisevich Aliaksei

**Comment by AndreySV** [comment]
ok. If I understand correctly, your wrapper is needed to check whether voltage is inside of range [lower_limit, upper_limit] and check_modbus did not support that check. Right? 
If so, the latest check_modbus has full support of nagios ranges and this task could be solved without any wrapper.

**Comment by leshcat** [comment]
The main point of the wrapper is to create always critical state whenever it over-reaches lower or upper limit. If modbus can (or could before, i am not sure) work like that out-of-box - that's great news and wrapper is indeed no longer required. :)

**Comment by AndreySV** [comment]
Yes, your case was not supported before, but it is now. Thanks for the blogpost and don't hesitate to contact me, if you have any issues with check_modbus. :)

**Comment by leshcat** [comment]
Thats great news!

I will reflect that at my post.

Take care,

Regards,

Dzenisevich Aliaksei
